custom/ewsghana/reminders/third_soh_reminder.py

```diff
@@ -1,14 +1,36 @@
+from corehq.apps.commtrack.models import SupplyPointCase
 from corehq.apps.locations.dbaccessors import get_web_users_by_location
 from corehq.apps.locations.models import SQLLocation
 from corehq.apps.reminders.util import get_preferred_phone_number_for_recipient
 from corehq.apps.users.models import CommCareUser
+from custom.ewsghana.reminders import THIRD_STOCK_ON_HAND_REMINDER, INCOMPLETE_SOH_TO_SUPER
+from custom.ewsghana.reminders.const import DAYS_UNTIL_LATE
 from custom.ewsghana.reminders.second_soh_reminder import SecondSOHReminder
-from custom.ewsghana.utils import send_sms, has_notifications_enabled
+from custom.ewsghana.utils import send_sms, has_notifications_enabled, report_status
 from dimagi.utils.couch.database import iter_docs
 
 
 class ThirdSOHReminder(SecondSOHReminder):
 
+    def get_message_for_location(self, location):
+        supply_point = SupplyPointCase.get_by_location(location)
+        if not supply_point:
+            return
```

Comment:
shouldn't this also return a tuple otherwise it will cause an error when trying to unpack the return value